packages/ui/components/Icon/Icon.tsx

```diff
@@ -11,6 +11,8 @@ export interface IconProps extends React.HTMLAttributes<HTMLOrSVGElement>{
   icon: IconType;
   size?: Size | number;
   color?: Color;
+  strokecolor?: Color;
+  storkethickness?: Size;
```

Comment:
Slight spelling error here. Also can we capitalize? `strokeThickness`?

Reply:
@tristanMatthias  I think it was the husky npm module that I was lacking, now I have it installed. Spelling mistakes are done as well.